### ISSUE TITLE
🚀 Don't escape unicode chars in the otput

### ISSUE THREAD

**Issue by eugenesvk** [dandavison/delta#1477]
When using some special chars in file names I get escaped ugliness in the output
```
delta '~/Dev/1/file—2.txt' '~/Dev/1/file—1.txt'

~/Dev/1/file\342\200\2242.txt ⟶   ~/Dev/1/file\342\200\2241.txt
```

Is it possible to just leave the symbols as is?
Sorry if I've missed some config, tried searching 'unicode' and 'escape' in the help files

**Comment by Freed-Wu** [comment]
Similar problem for other unicode characters. The above is `git diff`, the below is `delta`:

![Screenshot from 2023-11-11 01-52-17](https://github.com/dandavison/delta/assets/32936898/50ea8dd3-62b2-4f99-ace4-a2507f20515d)

**Comment by Freed-Wu** [comment]
Even the mid line cannot be aligned.

![Screenshot from 2023-11-11 01-54-02](https://github.com/dandavison/delta/assets/32936898/9f2e2aaa-a7fb-432e-80fa-749fe4954e38)

**Comment by opennomad** [comment]
I did some more digging and since the docs mention highlighting is based on [bat](https://github.com/sharkdp/bat), I took a look there and found [issue 2578](https://github.com/sharkdp/bat/issues/2578), which looks similar.

The recommendations there worked for me. I first tested the same file with `bat` and also saw the `<U+....>`. Disabling the paging showed the correct symbol, which points to `less` as the culprit.

So I set `export LESSUTFCHARDEF=E000-F8FF:p,F0000-FFFFD:p,100000-10FFFD:p` and both `bat` and `delta` started showing the symbols.

**Comment by th1000s** [comment]
That is git which escapes the paths, the related move is fixed in #1840. To not quote it set

    [core]
         quotepath = true

The (unrelated) `LESSUTFCHARDEF` issue with newer `less` versions is fixed in 0.18.0 see [this comment on #1616](https://github.com/dandavison/delta/issues/1616#issuecomment-2219749465) for more.
